src/common/eeprom.cpp

```diff
@@ -469,6 +469,16 @@ const char *eeprom_get_var_name(enum eevar_id id) {
     return "???";
 }
 
+bool eeprom_find_var_by_name(const char *name, enum eevar_id *var_id_out) {
+    for (int i = 0; i < (int)EEPROM_VARCOUNT; i++) {
+        if (strcmp(eeprom_map[i].name, name) == 0) {
+            *var_id_out = (enum eevar_id)i;
+            return true;
+        }
+    }
+    return false;
+}
```

Comment:
The eeprom's interface is written in C...

Reply:
I see. That's sad. I've also hit few places where it would really benefit by turning stuff to more C++-style, but that would ripple down to several still C places that use it :-(.